generators/server/templates/src/main/java/package/config/_MetricsConfiguration.java

```diff
@@ -12,7 +18,11 @@
 import fr.ippon.spark.metrics.SparkReporter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.boot.actuate.autoconfigure.ExportMetricReader;
```

Comment:
these new imports should be only when `<%_ if (applicationType == 'microservice' || applicationType == 'gateway') { _%>`